internal/platform/redis/client.go

```
@@ -0,0 +1,145 @@
+}
+
+func (c *Client) do(ctx context.Context, args ...string) ([]byte, error) {
+	var dialer net.Dialer
+	ctx, cancel := context.WithTimeout(ctx, c.dialTimeout)
+	defer cancel()
+
+	conn, err := dialer.DialContext(ctx, "tcp", c.addr)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)